pages/linux/pacdiff.md

```diff
@@ -0,0 +1,32 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> More information: <https://manned.org/pacdiff>.
+
+- Look for files that need maintenance:
+
+`pacdiff`
+
+- Use sudo and sudoedit to remove and merge files:
+
+`pacdiff --sudo`
+
+- Store the original version of any file you `(O)verwrite with` as a `.bak`:
+
+`pacdiff --backup`
```

Comment:
Does backup need sudo?

Reply:
I just checked and you need either `sudo` or `-s` when changing files.

Reply:
Is sudo prefered over `-s`?

Reply:
Only use sudo where it could be useful and is required. And in this case since it has the feature built in we should use it, so the `-s` IMO is the best option here for the commands that need root.